### PR TITLE
Added route with sub-routes for Dependant

### PR DESCRIPTION
**Pull request by FuriousLlama**
# Fixes or Implements [AB#9422](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/9422) & [AB#9424](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/9424)
* [ ] Enhancement
* [ ] Bug
* [ ] Documentation

## Description

Added the subroutine handling for dependant controller

## Testing

* [ ] Unit Tests Updated
* [ ] Functional Tests Updated
* [x] Not Required

### CODE REVIEW

File: Apps/WebClient/src/Server/Controllers/DependentController.cs [hunk 1]
```diff
@@ -32,7 +32,7 @@ namespace HealthGateway.WebClient.Controllers
     /// </summary>
     [Authorize]
     [ApiVersion("1.0")]
-    [Route("v{version:apiVersion}/api/[controller]")]
+    [Route("v{version:apiVersion}/api/UserProfile/")]
```
Comment (tiagograf-AOT) [inline Apps/WebClient/src/Server/Controllers/DependentController.cs:35]: It would be nice if there was a way of injecting typed controller in here!

**Review by tiagograf-AOT — DISMISSED** [review]

File: Apps/WebClient/src/Server/Controllers/DependentController.cs (hunk 1 above)
Comment (FuriousLlama) [reply Apps/WebClient/src/Server/Controllers/DependentController.cs:35]: Yeah! I tried a couple of things but no luck

File: Apps/WebClient/src/ClientApp/src/services/restDependentService.ts [hunk 2]
```diff
@@ -39,7 +39,7 @@ export class RestDependentService implements IDependentService {
             }
             this.http
                 .post<RequestResult<AddDependentRequest>>(
-                    `${this.BASE_URI}/`,
+                    `${this.BASE_URI}/UserProfile/{hdid}/Dependent`,
```
Comment (leobcgov) [inline Apps/WebClient/src/ClientApp/src/services/restDependentService.ts:42]: should it be ${hdid}

**Review by leobcgov — CHANGES_REQUESTED** [review]

**Review by leobcgov — DISMISSED** [review]
I have a comment on this PR but don't want to hold it off as I will be off tomorrow (Friday)

**Review by tiagograf-AOT — APPROVED** [review]

**Review by leobcgov — APPROVED** [review]

**Pull request merged**